src/PerfCounters.cc

```diff
@@ -1118,4 +1125,166 @@ Ticks PerfCounters::read_ticks(Task* t, Error* error) {
   return ret;
 }
 
+
+class BpfAccelerator {
+public:
+  static std::shared_ptr<BpfAccelerator> get_or_create();
```

Comment:
I was thinking we could just create one `BpfAccelerator` in `ReplaySession` and copy the reference when we clone `ReplaySession`s so we don't need a static variable here.

Reply:
I'm not convinced this is a great idea. It means moving `BpfAccelerator` into the header so `ReplaySession` can get at it. Is that really better than a static singleton?